src/platform/web/ui/general/LazyListView.js

```diff
@@ -0,0 +1,128 @@
+import {mountView} from "./utils.js";
+import {insertAt, ListView} from "./ListView.js";
+
+class Range {
+    constructor(start = 0, end = 0) {
+        this.start = start;
+        this.end = end;
+        this._expanded = false;
+    }
+
+    _onInitialExpand() {
+        if (this._expanded) { return; }
```

Comment:
What does expand mean here? Move the window of the range to react to scrolling? Is it inspired [on this](https://github.com/matrix-org/matrix-react-sdk/blob/9d6d8fc666855ca0c06c71ccb30c74ac4fc8fd12/src/components/views/elements/LazyRenderList.js#L39)? It seems to do something somewhat different though.